banks-server/src/banks_server.rs

```diff
@@ -161,6 +165,20 @@ fn verify_transaction(
     }
 }
 
+fn verify_versioned_transaction(
+    transaction: &VersionedTransaction,
+    _feature_set: &Arc<FeatureSet>,
+) -> transaction::Result<()> {
+    if let Err(err) = transaction.verify_and_hash_message() {
+        Err(err)
+    // TODO: Can't verify precompiles without looking up address map accounts...
+    //} else if let Err(err) = transaction.verify_precompiles(feature_set) {
+    //    Err(err)
```

Comment:
Check if verify_precompiles is possible because program accounts are guaranteed to be in the static list?